web/main.py

```diff
@@ -5,12 +5,12 @@
 import logging
 import mimetypes
 import os
-from flask import Flask, abort, request, Response, stream_with_context
 
-from cpg_utils.cloud import read_secret, is_member_in_cached_group
-import google.cloud.storage
 import google.auth.transport.requests
+import google.cloud.storage
 import google.oauth2.id_token
+from cpg_utils.cloud import is_member_in_cached_group, read_secret
+from flask import Flask, Response, abort, request, stream_with_context
```

Comment:
total nit, but is this post `isort`? flask should be before internal libs, but it won't be recognising cpg-utils as internal

Reply:
Got this incoming soon: https://github.com/populationgenomics/cpg-python-template-repo/pull/12#issuecomment-1881917977
That should sort our isort funs